requirements/constraints.txt

```diff
@@ -40,7 +40,7 @@ drf-yasg<1.17.1
 # The team that owns this package will manually bump this package rather than having it pulled in automatically.
 # This is to allow them to better control its deployment and to do it in a process that works better
 # for them.
-edx-enterprise==3.6.0
+edx-enterprise==3.6.2
```

Comment:
hmmmmmm surprised someone else didn't put in the 3.6.1 change here... I'd be careful with this even when the new feature merging block ends. Double-check there aren't PRs outstanding for 3.6.1 or look at the changes and see if it could have database changes/something else that might be problematic back to back.

Reply:
I was partially assuming that whoever made the 3.6.1 changes were abiding by the same "please do not release new features" request that I am, so the plan was to wait until we can release again, double check if 3.6.1 has been released and change accordingly.

Reply:
yeah that's why I said double check the state of things. Sounds like a good plan. :)